### PR TITLE
Request status certificate

### LINKED ISSUE #297: Create LaTeX template for certificates

### PR DESCRIPTION
**Pull request by luksan47**
Fixes #297 #237 
Added status certificate document. Users can now download the pdf or request one from the secretariat. Upon request, the secretaries will be notified via email and will be redirected to the user's certificate.

### CODE REVIEW

File: resources/lang/hu/document.php [hunk 1]
```diff
@@ -5,19 +5,23 @@
...
+    'missing_educational_info' => 'A tanulmányi adataid hiányoznak a dokumentum kitöltéséhez. Kérj segítséget egy rendszergazdától.',
     'missing_items' => 'Még nem adtad meg a tárgyakat, amiket behoznál.',
     'missing_personal_info' => 'A személyes adataid hiányoznak a dokumentum kitöltéséhez. Kérj segítséget egy rendszergazdától.',
     'not_report' => 'Nem kell bejelenteni:',
     'print' => 'Nyomtatás',
     'report' => 'Be kell jelenteni:',
+    'request' => 'Igénylés',
     'serial_number' => 'Sorozatszám (számítógépeknél)',
+    'status-cert' => 'Tagsági igazolás',
+    'successful_request' => 'Sikeres igénylés. Az igazolást hamarosan megtalálhatod a titkárásgon.',
```
Comment (kdmnk) [inline resources/lang/hu/document.php:26]: typo: `titkárságon`

File: resources/views/emails/status_certificate_request.blade.php [hunk 2]
```diff
@@ -0,0 +1,7 @@
+@component('mail::message')
+<h1>@lang('mail.dear') {{ $recipient }}!</h1>
+<p>
+@lang('mail.status_cert_request', ['user' => $user]) <a href="{{ $url }}">@lang('mail.show')</a>
```
Comment (kdmnk) [inline resources/views/emails/status_certificate_request.blade.php:4]: You can use the button component for links (see `approved_registration` email blade).

File: resources/views/latex/status-cert.blade.php [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+\usepackage{multirow}
+
+\usepackage{caption}
+\usepackage[letterspace=300]{microtype}
+
+\usepackage{atbegshi}
+\usepackage{tikz}
+\usepackage[left=2cm, right=2cm, top=5cm, bottom=1cm]{geometry}
+\definecolor{darkblue}{rgb}{0.2, 0.2, 0.7}
+\title{Igazolás önkéntes tevékenységről\vspace{-1.0cm}}
```
Comment (kdmnk) [inline resources/views/latex/status-cert.blade.php:15]: Not `Tagsági igazolás`? or sth like that

**Review by kdmnk — APPROVED** [review]
left some comments

**Comment by kdmnk** [comment]
Also, please supplement the description on document page with some comments about requiring documents from the secretariat.

File: resources/views/latex/status-cert.blade.php [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+	#3
+\end{center}
+}
+
+\begin{document}
+\maketitle
+
+Alulírott Horváth László, az ELTE Eötvös József Collegium igazgatója, hivatalosan igazolom, hogy {{ $name }} (Neptun-kód: {{ $neptun }}) az ELTE Eötvös József Collegium tagja {{ $from }}. szeptemberétől.
+
+A tagság érvényességének befejezése: {{ $until }}
```
Comment (kdmnk) [inline resources/views/latex/status-cert.blade.php:83]: How valid is this? Maybe we can change it to `várható befejezése`

Comment (luksan47) [reply resources/views/latex/status-cert.blade.php:83]: I got this template from the secretariat, this has been used for quite some time now. I don't really like it either, but can't think of better.

**Pull request merged**